time cam_frame
time start_proc
time end_proc
duration proc_time
duration cam_frame_delay
